experiments/wasm-eel/Flexi, martin + geiss - dedicated to the sherwin maxawow.json

```diff
@@ -0,0 +1,57 @@
+{
+  "baseVals": {
+    "rating": 3,
+    "gammaadj": 1.56,
+    "decay": 1,
+    "echo_zoom": 0.362,
+    "echo_orient": 1,
+    "wave_thick": 1,
+    "wave_brighten": 0,
+    "darken": 1,
+    "wave_a": 0.001,
+    "wave_scale": 1.599,
+    "wave_smoothing": 0,
+    "wave_mystery": -0.5,
+    "modwavealphastart": 2,
+    "modwavealphaend": 2,
+    "warpscale": 0.107,
+    "zoomexp": 0.1584,
+    "fshader": 1,
+    "warp": 0.01,
+    "wave_r": 0.51,
+    "wave_g": 0.5,
+    "ob_size": 0,
+    "ob_a": 1,
+    "ib_r": 0,
+    "ib_g": 0,
+    "ib_b": 0,
+    "ib_a": 1,
+    "mv_x": 64,
+    "mv_y": 48,
+    "mv_l": 0.5,
+    "mv_r": 0,
+    "mv_g": 0,
+    "mv_b": 0,
+    "mv_a": 0
+  },
+  "shapes": [
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } }
+  ],
+  "waves": [
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } },
+    { "baseVals": { "enabled": 0 } }
+  ],
+  "init_eqs_str": "a.d=0;a.cx1=0;a.y3=0;a.y1=0;a.cy1=0;a.x1=0;a.x3=0;a.dir=0;a.r=0;a.x2=0;a.y2=0;",
+  "frame_eqs_str": "a.ib_r=.3*Math.sin(5*a.time)+.7;a.ib_g=.3*Math.sin(4*a.time)+.3;a.ib_b=.5*Math.sin(4*div(a.time,3))+.5;a.wave_r=1-a.ib_r;a.wave_g=1-a.ib_g;a.wave_b=1-a.ib_b;a.wave_x=.5+.3*Math.sin(3*a.time);a.wave_y=.5+.3*Math.cos(2.187*a.time);",
+  "pixel_eqs_str": "a.r=div(a.bass,4);a.cx1=.5+.2*Math.sin(.618*a.time);a.cy1=.5+.2*Math.cos(1.618*a.time);a.d=sqrt((a.x-a.cx1)*(a.x-a.cx1)+(a.y-a.cy1)*(a.y-a.cy1));a.dir=a.bass*(a.r*a.r-a.d*a.d)*.3;a.x1=.00001<Math.abs(above(a.d,a.r))?0:Math.sin(a.y-a.cy1)*a.dir;a.y1=.00001<Math.abs(above(a.d,a.r))?0:-Math.sin(a.x-a.cx1)*a.dir;a.cx1=.5+.3*Math.sin(2.618*a.time);a.cy1=.5+.3*Math.cos(3.14*a.time);a.d=sqrt((a.x-a.cx1)*(a.x-a.cx1)+(a.y-a.cy1)*(a.y-a.cy1));a.dir=-a.mid*(a.r*a.r-a.d*a.d)*.3;a.x2=.00001<\nMath.abs(above(a.d,a.r))?0:Math.sin(a.y-a.cy1)*a.dir;a.y2=.00001<Math.abs(above(a.d,a.r))?0:-Math.sin(a.x-a.cx1)*a.dir;a.cx1=.5+.4*Math.sin(2.618*-a.time);a.cy1=.5+.4*Math.cos(1.14*-a.time);a.d=sqrt((a.x-a.cx1)*(a.x-a.cx1)+(a.y-a.cy1)*(a.y-a.cy1));a.dir=-a.treb*(a.r*a.r-a.d*a.d)*.3;a.x3=.00001<Math.abs(above(a.d,a.r))?0:Math.sin(a.y-a.cy1)*a.dir;a.y3=.00001<Math.abs(above(a.d,a.r))?0:-Math.sin(a.x-a.cx1)*a.dir;a.dx=a.x1+a.x2+a.x3;a.dy=a.y1+a.y2+a.y3;",
+  "init_eqs_eel": "",
```

Comment:
How come there's an `"init_eqs_str"` but no `"init_eqs_eel"`?

Reply:
I have to manually set the user custom variables to `0` in the initialization, but the WASM interpreter handles that for us :D 👍